build.xml

```diff
@@ -809,6 +811,7 @@
       <classpath>
         <fileset dir="${build.plugins}" >
           <include name="**/*.jar"/>
+          <exclude name="any23/javax.annotation-api*.jar"/>
```

Comment:
Ok. Nice, this solves the remaining "Multiple sources of package comments found" warning (see #568).

Reply:
Hi @sebastian-nagel in short no I am not able to reproduce.
```% ant clean runtime
...
  [javadoc] Standard Doclet version 1.8.0_221
  [javadoc] Building tree for all the packages and classes...
  [javadoc] Building index for all the packages and classes...
  [javadoc] Building index for all classes...
     [copy] Copying 1 file to /Users/lmcgibbn/Downloads/nutch/build/docs/api/org/apache/nutch/plugin/doc-files
     [copy] Copying 1 file to /Users/lmcgibbn/Downloads/nutch/build/docs/api/resources
     [copy] Copying 1 file to /Users/lmcgibbn/Downloads/nutch/build/docs/api/resources

BUILD SUCCESSFUL
Total time: 38 seconds
```